bot/utils/decorators.py

```diff
@@ -112,10 +148,12 @@ def decorator(callable_: Callable) -> Callable:
         # Functions decorated as commands are turned into instances of `Command`
         if isinstance(callable_, Command):
             log.debug(f"Command {callable_.qualified_name} will be locked to {human_months(allowed_months)}")
-            actual_deco = in_month_command(*allowed_months)
+            actual_deco = in_month_command(*allowed_months, roles=roles)
 
         # D.py will assign this attribute when `callable_` is registered as a listener
         elif hasattr(callable_, "__cog_listener__"):
+            if roles is not None:
```

Comment:
This raises an error even when `roles` is not provided because the default is not None.
```suggestion
            if roles:
```

Reply:
Oh right, it was None at first haha